concourse/pipeline/pr.yml

```
@@ -14,7 +17,11 @@
 #@   "res_map": res_map,
 #@   "gpdb_src": "gpdb6_src",
 #@   "trigger": pr_trigger(res_map),
-#@   "confs": [centos7_gpdb6_conf()]
+#@   "confs": [
+#@     centos6_gpdb6_conf(),
+#@     centos7_gpdb6_conf(),
+#@     rhel8_gpdb6_conf(),
+#@     ubuntu18_gpdb6_conf()]
```

```suggestion
#@     ubuntu18_gpdb6_conf()
    ]
```
Would this be better? only need to change one line if need to add one conf.